rust/src/lib.rs

```diff
@@ -2,3 +2,7 @@
 
 mod error;
 mod wrapper;
+
+trait DeepClone {
+    fn deep_clone(&self) -> Self;
+}
```

Comment:
I believe we don't need this trait for now, as it's currently used to add a method to a single struct, `Format`. It might be more appropriate to introduce this trait later if we need to deeply clone a member variable by invoking this method externally.

Reply:
Fixed.